zerver/tests/test_docs.py

```diff
@@ -357,6 +357,13 @@ def test_integration_doc_endpoints(self) -> None:
             not in (image_path := os.path.relpath(os.path.join(root, file), directory))
         }
 
+        # The integration doc and the screenshot images are in different repos,
```

Comment:
This is specifically for the `PYTHON_API_INTEGRATIONS`, correct? I wonder if it'd be helpful to mention that here somewhere as well. Just to clarify a bit why and what we're adding exceptions for.